public/icons/Loading.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
I meant to add the icon as an SVG file here and then import as a component inside the component where it should be used.